components/EventsView.tsx

```diff
@@ -138,14 +138,17 @@ const EventsView: React.FC<EventsProps> = ({ material, events }) => {
             <Timeline.Item key={event.id}>
               <Timeline.Point />
               <Timeline.Content>
-                <Timeline.Time className="flex justify-between">
+                <div className="flex justify-between">
                   <Link href={`/event/${event.id}`} className="text-gray-600 dark:text-gray-200">
-                    {getFormattedDate(event.start)}
+                    <Timeline.Time dateTime={event.start.toISOString()}>{getFormattedDate(event.start)}</Timeline.Time>
                   </Link>
                   {isAdmin && (
                     <Stack direction="row">
                       <Tooltip title="Duplicate Event">
                         <MdContentCopy
+                          role="button"
+                          tabIndex={0}
+                          aria-label="Duplicate Event"
```

Comment:
```suggestion
                          aria-label={`Duplicate event: ${event.id}`}
```

Presumably this since you are gonna have dozens of identically labelled buttons?